ll-token-field.html

```diff
@@ -110,18 +167,32 @@
         e.stopPropagation();
         var newTag = input.value;
         input.value = '';
-        this.tags = this.tags.concat(newTag);
+        this.push('tags', newTag);
+        this.tagChanged();
         input.focus();
       }
       else if (e.keyCode === 8 && input.value === '') { // Backspace
```

Comment:
Added a regex check that should fail the input if it is just whitespace. I also added a test to validate the failure.